<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamWidget</class>
 <widget class="QWidget" name="ParamWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="paramSel"/>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="paramStack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="NoPWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>No parameter set loaded</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="PWParam" name="PWWidget"/>
     <widget class="CPParam" name="CPWidget"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PWParam</class>
   <extends>QWidget</extends>
   <header>qt/params/pwparam.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CPParam</class>
   <extends>QWidget</extends>
   <header>qt/params/cpparam.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
